homeassistant/components/knx/const.py

```diff
@@ -129,6 +129,13 @@ class ColorTempModes(Enum):
     RELATIVE = "5.001"
 
 
+class FanZeroMode(Enum):
+    """Enum for setting the fan zero mode."""
+
+    OFF = 1
+    AUTO = 2
```

Comment:
```suggestion
class FanZeroMode(StrEnum):
    """Enum for setting the fan zero mode."""

    OFF = FAN_OFF
    AUTO = FAN_AUTO
```
Import FAN_OFF and FAN_AUTO from the climate component (alternatively move this enum to knx.climate). 
We can then replace `climate.py`
```
fan_zero_mode = config[ClimateSchema.CONF_FAN_ZERO_MODE]
self.fan_zero_value = FAN_OFF if fan_zero_mode == FanZeroMode.OFF else FAN_AUTO
```
with
```
self.fan_zero_mode = config[ClimateSchema.CONF_FAN_ZERO_MODE]
```

In the schema this could even be used like
```
vol.Coerce(FanZeroMode)
# instead of 
vol.All(vol.Upper, cv.enum(FanSpeedMode)),
```

StrEnum should be preferred over plain Enum in HA integrations. Those plain Enums we have are more or less pre-py-3.11 things.

Reply:
fixed